index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" ahref="style.css">
+  </head>
```

Comment:
Also, add other meta tags to give you browser more information about your website. 
ex. 
`
  <meta charset="UTF-8">
`
`
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
`
` <title>Document</title>
`